packages/build/src/plugins_core/functions/index.ts

```diff
@@ -38,7 +38,7 @@ const eventTriggeredFunctions = new Set([
   'identity-login',
 ])
 
-const validateCustomRoutes = function (functions: Awaited<ReturnType<typeof zipFunctions>>) {
+const validateCustomRoutes = function (functions: FunctionResult[]) {
```

Comment:
Not really related to the metadata file, but it's good to use the imported types from zip-it-and-ship-it.